lib/ability.rb

```diff
@@ -19,6 +19,10 @@ def initialize(user, resource, controller)
         can :manage, AccountPriceGroupMember
       else
         can :manage, :all
+        cannot [:manage_accounts, :manage_billing], Facility.cross_facility
+        if resource.blank? || resource.is_a?(Facility) && resource.cross_facility?
```

Comment:
This is another spot that might be more clear with a null object. The resource might be blank if `current_facility` is nil.